README.md

```diff
@@ -47,12 +47,12 @@ mailing list, if preferred.
 ### Setting up for development
 
 #### Development Compiler
-The version of the compiler recommended for development is 1.43. Other
+The version of the compiler recommended for development is 1.44.1. Other
 versions of the compiler may disagree with the CI tasks on some points,
 so should be avoided.
 
 #### Building
-Stratisd requires Rust 1.43+ and Cargo to build. These may be available via
+Stratisd requires Rust 1.44.1+ and Cargo to build. These may be available via
```

Comment:
This is the line we want to leave alone.

Reply:
Ok; I'm going to close this PR, and re-open, holding this line at "1.43+".